percent_box.js

```diff
@@ -114,11 +114,17 @@ class PercentBoxElement extends HTMLElement {
       this._pbox.update(data);
       toggleSVGContainerAndIndicator.call(this, false);
       broker.addEventListener('stats-stream-data', (evt) => {
-        const stats = evt.detail;
-        const val = stats[this.percentKey];
-        const total = stats[this.totalKey];
-        data = [ val, total - val ];
-        this._pbox.update(data);
+        const stats = evt.detail.stats;
+        // Determine if the values come from index mapped reads or sampled mapped reads
+        if (this.percentKey === "calculated_mapped_reads" && stats.calculated_mapped_reads !== stats.mapped_reads &&
```

Comment:
As I mentioned before, we don't want this code here because it makes the percent box so that it's not generic anymore. We only want to handle generic data. We don't want special cases for our data.